src/lib/mappers/charge-module.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+const { truncate, identity, pick } = require('lodash');
+const invoiceAccountMapper = require('../../lib/mappers/invoice-account');
+const { combineAddressLines, getAddressObjectFromArray } = require('./lib/helpers');
+
+/**
+ * @module maps service models to charge module expected schema
+ */
```

Comment:
I think that these functions should live in the invoice accounts mappers file (`src/lib/mappers/invoice-account.js`)

Reply:
I'd like to keep this in its own file because that's what we did for Notify's mappers. Would that be OK?

Reply:
It's hard to know what the correct convention is over such a large and sprawling code base. We do have `modelToChargeModule` mappers for `Batch` and `Transaction` models, so I saw this as the equivalent to that for the `InvoiceAccount` and `Address` models as that is where I would look for them